src/reduced_order/pod_basis.cpp

```diff
@@ -0,0 +1,261 @@
+#include <deal.II/lac/sparse_matrix.h>
+#include "pod_basis.h"
+#include <deal.II/fe/mapping_q1_eulerian.h>
+
+namespace PHiLiP {
+namespace ProperOrthogonalDecomposition {
+
+template <int dim>
+POD<dim>::POD(std::shared_ptr<DGBase<dim,double>> &_dg)
```

Comment:
Same as the previous comment on the underscore notation for inputs

Reply:
Fixed in most recent commit.